sdk/src/test/scala/com/horizen/account/receipt/ReceiptFixture.scala

```diff
@@ -42,7 +42,7 @@ trait ReceiptFixture {
     } else {
       new Array[Byte](0)
     }
-    val consensusDataReceipt = new EthereumConsensusDataReceipt(txType, 1, BigInteger.valueOf(1000), logs, new Array[Byte](256))
+    val consensusDataReceipt = new EthereumConsensusDataReceipt(txType, 1, BigInteger.valueOf(1000), logs, new LogsBloom())
```

Comment:
here `new LogsBloom()` is not consistent to `logs`. Use the same constructor without `...new LogsBloom()...`